### DISCUSSION TITLE
Google Pixel Watch and Standalone Dexcom G6

### DISCUSSION THREAD

**Discussion by BrittT1** [NightscoutFoundation/xDrip#2874] [Q&A]
Hi there - Have been following the instructions for enabling pixel watch (just purchased last week) to serve as the primary receiver of Dexcom data but can't seem to get it to be successful. My phone is a Pixel 6.  My goal is to be able to leave my phone at home and still get my BGs. I've gotten to the point where the event logs show the correct entries but then it stalls out. Can anyone help? I am in the Atlanta area and am using G6. I did search the discussions for this issue but not sure what the results are when I type in 'pixel'....ack!

**Comment by Navid200** [comment]
Can you share the instructions you have for making a Pixel watch a stand-alone collector from a G6?

**Reply by BrittT1** [reply]
Yes! https://xdrip.readthedocs.io/en/latest/smartwatch/wear/

**Reply by Navid200** [reply]
@psonnera 
How valid is that table?  It claims that some watches can collect stand-alone once every 10 minutes from G6.  Is that a valid statement today?

**Comment by psonnera** [comment]
Best case every 10 minutes is accurate. Still, I don't have much feedback on how many still suceed. Pixel I believe nobody. I don't know how much of the G6 code modification/improvement was brought to the Wear extension...
